RFS/src/main/java/com/rfs/common/RestClient.java

```diff
@@ -38,34 +69,52 @@ public RestClient(ConnectionDetails connectionDetails) {
             });
     }
 
+    @WithSpan
```

Comment:
Need to double check if these work with Mono<?> - whereas they do work with completable futures.

Reply:
@gregschohn I've had a hard time getting started 'monitoring' the spans and so I'm going to call it quits on iterating here

Reply:
The largest unknown to me is if the addition of the extra telemetry instrumented could cause us issues around deployment or performance impacts